core/src/main/res/values/strings.xml

```diff
@@ -7,4 +7,5 @@
     <string name="kevin_no">No</string>
     <string name="kevin_dialog_exit_confirmation_title">Warning!</string>
     <string name="kevin_navigate_back_content_description">Navigate back</string>
+    <string name="kevin_processing">Processing…</string>
```

Comment:
What about localization, will we do it before the release?

Reply:
To avoid headaces I will just leave the progress bar. Should be more then enough especially that this is only an edge case fix.